docs/transition/index.md

```diff
@@ -37,3 +37,43 @@ Requests to the link of the format `https://rink.hockeyapp.net/api/2/apps/0873e2
 ## Feedback
 
 App Center doesn't support the HockeyApp Feedback functionality. For potential alternatives, see the [feedback guide](feedback.md).
+
+## FAQS
+
+#### Why are we retiring HockeySDK?
+
+In 2018, we announced the transition from HockeyApp to Visual Studio App Center, a single solution for continuously building, testing, releasing, and monitoring your apps. HockeyApp transitions are now complete, but we extended the retirement of the HockeySDK to November 13, 2020, to give you more time to move to App Center SDK. 
+
+#### What will happen after November 13th, 2020?
+
+All apps that still include the HockeySDK after November 13, 2020 will stop sending data to the App Center Portal. You will no longer receive Analytics and Diagnostics data for these versions, and users will no longer receive in-app updates. 
+
+#### What services will be impacted?
+
+All App Center services for which you are using HockeySDK will be impacted. This includes Diagnostics, Analytics, and Distribute in-app updates. 
+
+#### How can I know if my apps are using HockeySDK?
+
+To know if your app is using HockeySDK, you will have to either look into your codebase or check for dependencies in Gradle, Podfile, NuGet,etc
+
+#### Can I get an extension on the deadline?
+
+The November 13, 2020 extended retirement date is final and we will not be able to grant any extensions. 
+
+#### How can I migrate to the App Center SDK?
+
+We have created detailed App Center SDK migration guides for each of our supported platforms. Links to each of these guides are listed at the top of this page. 
+
+#### I have already migrated my app to the App Center SDK? Why did I still get the retirement announcement?
+
+If you have already migrated to App Center SDK, it is likely that few of your customers are still using the old versions of the app with HockeySDK. You will stop receiving data from these customers after November 13th, 2020 unless these customers upgrade to the app version with App Center SDK. 
+
+#### The App Center SDK is missing a feature that the HockeySDK provided. Will these features be added to App Center SDK before the retirement?
+
+The App Center team has spent the last couple of years closing the majority of feature parity gaps with the HockeySDK. While a small number of feature parity gaps still exist, we do not intend on investing further in closing these gaps.
```

Comment:
I'd extend this because it raises some questions about status of existing requests/issues from customers. Something like:
> The existing requests for adding missing feature that was implemented in HockeySDK will be triaged in general order along with other feature requests.

or explicitly write that we're going to close these requests.

Reply:
@gaknoll: Ivan raises a good point on the status of existing requests/issues. What are your thoughts on it?